backend/src/prompts/templates/intent.j2

```diff
@@ -18,6 +18,10 @@ Q: How much did I spend with Amazon this month?
 Response:
 {"query":"How much did I spend with Amazon this month?","user_intent":"sum amount spent","questions":[{"query":"How much did I spend with Amazon this month?","question_intent":"calculate total expenses","operation":"filter + aggregation","question_category": "data driven","parameters":[{"type":"company","value":"Amazon"}],"timeframe":"this month","aggregation":"sum","sort_order":"none"}]}
 
+Q: What are the mortgage rates offered by NatWest for first-time buyers?
+Response:
+{"query":"What are the mortgage rates offered by NatWest for first-time buyers.","user_intent":"find information online","questions":[{"query":"What are the mortgage rates offered by NatWest for first-time buyers.","question_intent":"retrieve information from web","operation":"online search and scrapping","question_category": "search online","parameters":[{"type":"topic","value":"mortgage rates","company":"Natwest"}],"sort_order":"descending"}]}

```

Comment:
Is it meant to be "scraping" instead of "scrapping" in the value for operation?

Reply:
Yes, of course. Have corrected the spelling.